pkg/utils/utils.go

```diff
@@ -0,0 +1,26 @@
+/*
+You may obtain a copy of the License at
```

Comment:
Looks like the licence text was truncated.  
```suggestion
/*
Copyright 2021.

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at
```